tests/app/questionnaire/test_schema_utils.py

```diff
@@ -209,20 +208,3 @@ def test_choose_question_to_display(question_variant_schema):
     )
 
     assert question_to_display["title"] == "Question 1, No"
-
-
-def test_get_values_for_key_ignores_variants():
```

Comment:
Due to the moving of get_values_for_key I have extended tests/app/questionnaire/conftest.py to cover these. They were also not working quite right, giving fake line coverage and hiding a bug. We never initially pass blocks into get_values_for_key only sections, only on the recursion of get_values_for_key did a block get passed, at which point it didn't pass the ignore_keys.

Reply:
They were lame duck tests and were only put in to hit this line. https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/questionnaire/schema_utils.py#L138. If the ignore_keys was corrected they weren't needed anyway, assuming the conftest was extended. The rest of the method is already tested with this https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/tests/app/questionnaire/test_questionnaire_schema.py#L203, although you could argue it isn't explicit. However now it is private it is difficult to get at, without an ignore. We could do it, but I would argue it should be extended to cover all scenarios